features.csv

```diff
@@ -1,4 +1,4 @@
-Feature,Debian,Ubuntu,Linux Mint,PopOS,MX Linux,Zorin OS,Gnome,Xfce,Cinnamon,Notes
+Feature,Debian,Ubuntu,Linux Mint,PopOS,MX Linux,Zorin OS,Arch,Gnome,Xfce,Cinnamon,Notes
```

Comment:
CSV file gets invalid through that. Please add a complete column with the features